website/server/controllers/api-v3/groups.js

```diff
@@ -514,6 +514,7 @@ async function _inviteByUUID (uuid, group, inviter, req, res) {
     }
 
     userToInvite.invitations.party = {id: group._id, name: group.name, inviter: inviter._id};
+    return userToInvite.save();
```

Comment:
Meant to put it at the bottom of the function, that was my mistake. I'll fix this.